### PR TITLE
Add Exercise01

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise01/Program.cs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+{ 
+    class Program
+    {
+        public static string ApiKey = "087498271007cb77b740b52dc9a1b4ad-us17";
+        public static string IdList = "2a02210045";
+        public static string IdCompaign = "90e5c4fc9c";
+
+        static void Main(string[] args)
+        {
+            Console.ReadKey();
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Please change:
- The program should allow user enter some content and an email address then send the email.
- Try to use retrofit.net library

**Review by voquanghoa — CHANGES_REQUESTED** [review]

File: Exercise01/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            var service = adapter.Create<IService>();
+
+            Console.Write("Send mail to email: ");
+            var str = Console.ReadLine();
+            var email = new Email { EmailAddress = str, Status = "subscribed" };
+            SimpleJson.CurrentJsonSerializerStrategy = new SerializerStrategy();
+            var response = service.AddEmail(IdList, email);
+            Console.WriteLine($"Add email: {response.ResponseStatus}");
+
+            /*Console.Write("Set content to mail: ");
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Send email ???????????????????????

Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+namespace Exercise01
+{ 
+    class Program
+    {
+        public static string ApiKey = "087498271007cb77b740b52dc9a1b4ad-us17";
+        public static string IdList = "2a02210045";
+        public static string IdCompaign = "fc0c0676fe";
+
+        static void Main(string[] args)
```
Comment (voquanghoa) [inline Exercise01/Program.cs:22]: Remove all unused methods

Comment (huynhngocminh2511) [reply Exercise01/Program.cs:22]: Done

**Review by voquanghoa — APPROVED** [review]

**Pull request merged**